### PR TITLE
Add SRGAN PyTorch Implementation

### PR DESCRIPTION
**Pull request by namangup**

### CODE REVIEW

File: generative_models/SRGAN_PyTorch/README.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+# PyTorch Implementation of SRGAN
+
+## Usage
+```bash
+$ python3 main.py
+```
+following are the arguments:
+```bash
```
Comment (nirmal-suthar) [inline generative_models/SRGAN_PyTorch/README.md]:
```suggestion
```
```

**Comment by nirmal-suthar** [comment]
You may change images of the results (grid are too congested and results are not properly highlighted)

**Comment by nirmal-suthar** [comment]
Move your folder to new folder named multimodal_models

**Review by nirmal-suthar — APPROVED** [review]

**Comment by namangup** [comment]
Done.

**Comment by nirmal-suthar** [comment]
Move to super_resolution folder, my bad

**Pull request merged**